src/wormhole/_rendezvous.py

```diff
@@ -61,6 +62,54 @@ def buildProtocol(self, addr):
         return proto
 
 
+def mint_hashcash(bits, resource):
+    "1:6:210623:arbitrary string::9WrCxB1SdCBOM3i5:000005"
+    counter = 0
+    timestamp = "210623"
```

Comment:
Since the server provides the `resource` part in a challenge-response design, checking the time stamp is not actually required for this to work. Nevertheless, I'd suggest you to include a current time anyways because the server side might use an off-the-shelf library that checks it regardless.